client/l2tp_client.c

```diff
@@ -963,6 +984,15 @@ void context_process(l2tp_context *ctx)
       }
       break;
     }
+    case STATE_GET_USAGE: {
+      // Get usage information if available
+      if (!is_timeout(&ctx->timer_usage, 2))
+        context_send_packet(ctx, CONTROL_TYPE_USAGE, "UUUUUUUU", 8);
+      else
+        /* time out. The broker might not support usage. ignore it. */
```

Comment:
Can we use consistent comments (in this case `//`)? And start them with a capital letter.